wrappers/java/src/main/java/tech/pegasys/era/tools/EraNetworkConfig.java

```diff
@@ -0,0 +1,309 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.era.tools;
+
+import tech.pegasys.era.*;
+import org.web3j.crypto.Credentials;
+import org.web3j.protocol.Web3j;
+import org.web3j.protocol.http.HttpService;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.nio.charset.StandardCharsets;
+import java.util.*;
+
+/**
+ *  A tool to auto config and deploy a network of Eras
+ *
+ * When using this program with tag --deploy-config,
```

Comment:
This documentation would be best moved into README.md (possibly the root read me).